### DISCUSSION TITLE
controlling opam installation directories for components

### DISCUSSION THREAD

**Discussion by bchill** [ocaml/opam#5361] [Q&A] [answered]
Hello,

I'd like to be able to install ocaml versions in /opt/ocaml/<version>/{bin,lib,etc,...} (as opposed to ~/.opam/<version>/...)

It's not clear to me if I need to use some combination of switch and install, and with which options.

If someone can point me in the right direction, I'd appreciate that.

Thanks!

Brian

**Comment by kit-ty-kate** [comment, marked as answer]
You can do this by simply exporting the following environment variable:
```
export OPAMROOT=/opt/ocaml
```
This is mentioned in the manual (https://opam.ocaml.org/doc/Manual.html#opam-root) and in the man-page/--help. If you think of an improvement to make this more clear, feel free to contribute.
